### ISSUE TITLE
Bug: Lured Pokestops at wrong location and won't ever go away

### ISSUE THREAD

**Issue by bellowsj** [AHAAAAAAA/PokemonGo-Map#2073]
When I enable pokestops I get three bogus pokestops near me that appear to be lured. These pokestops do not exist in the game.

## Expected Behavior

These 3 lured pokestops should not be showing on the map.

## Current Behavior

I get the three lured pokestops that should not be there.

## Possible Solution

My location is: 40.529238, -111.475289. The 3 lured pokestops near me should not there. If you are not seeing those 3 lured pokestops then maybe I need to clear my db? Any instructions for how to do that?

## Steps to Reproduce (for bugs)

Run the app and set your location to 40.529238, -111.475289. Scan out 5 or 6 steps and see if you see the 3 lured pokestops near me.

1.
2.
3.
4.

## Your Environment

* Version used: Develop branch. pulled yesterday (7/24)
* Environment name and version (e.g. Python 2.7): python 2.7.12
* Operating System and version (desktop or mobile): Windows 7 and Chrome 51.

Here are a few screenshots of the issue:

![broken_pokestop_lured](https://cloud.githubusercontent.com/assets/380933/17116074/55fbbe60-5273-11e6-872b-80ccee21679c.png)

![broken_pokestop2_lured](https://cloud.githubusercontent.com/assets/380933/17116122/8b08caee-5273-11e6-824b-110f25f8f7dd.png)

![broken_pokestop3_lured](https://cloud.githubusercontent.com/assets/380933/17116126/8f73c0c0-5273-11e6-9939-6e35aea44e21.png)

**Comment by DeviantEng** [comment]
Are you running the web server with the `-m` switch?  While undocumented, `-m` runs the webserver, no scanner, but inserts dummy data.  I spent a good 12 hours over the weekend trying to figure out why I had the same issue.  Instead of `-m`, you should use `-os`.

**Comment by invisiblek** [comment]
have you tried nuking your database?

**Comment by DeviantEng** [comment]
I didn't have to recreate my db.  Just had to stop any running processes that were using `-m`, and restart with `-os` instead.

**Comment by sfjuocekr** [comment]
To remove the DB simply:

rm pogom.db

**Comment by bellowsj** [comment]
Ah ok. I didn't know about the `-os` switch so I was using `-m` for my webserver and then running 4 scanners with `-ns`. But switching it to just `-os` didn't solve my issue alone--I had to remove the db as well. Thanks for the help guys. One last question, is there a way to show just the lured pokestops? I know we used to be able to do that with a command line switch a while ago but now that everything is in the gui I don't see a way to show only lured pokestops.
